disdrodb/tests/test_check_config.py

```diff
@@ -0,0 +1,29 @@
+import os
+import pytest
+
+
+from disdrodb.l0 import check_configs
+from disdrodb.l0.standards import available_sensor_name
+
+PATH_TEST_FOLDERS_FILES = os.path.join(os.path.dirname(os.path.realpath(__file__)), "pytest_files")
+
+
+# Test that all instruments config files are valid regarding the bin consistency
+@pytest.mark.parametrize("sensor_name", available_sensor_name())
+def test_check_bin_consistency(sensor_name):
+    # Test that no error is raised
+    assert check_configs.check_bin_consistency(sensor_name) is None

```

Comment:
In the tests, you always imported the module and then use `<module>.<function>`
Is there any reason to not import the tested function at the top of the file or at the starting line of the test functions?
Importing at the start would also reduce the text length and verbosity ...

Reply:
I'll correct that. thx